app/notifications/process_notifications.py

```diff
@@ -87,3 +89,9 @@ def send_notification_to_queue(notification, research_mode, queue=None):
     current_app.logger.info(
         "{} {} created at {}".format(notification.notification_type, notification.id, notification.created_at)
```

Comment:
I've moved this log message to the persist method and changed this to talk about sending it to the queue.